source/css/sass/mixins/_typography.scss

```diff
@@ -67,6 +67,25 @@
   vertical-align: middle;
 }
 
+@mixin  text-scripts {
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline;
+}
+
+@mixin _font-variant-position($position) {
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline;
```

Comment:
It might be necessary to super- / -sub script something which doesn't have that markup. Feels like a corner case, but we probably shouldn't rule it out.